### ISSUE TITLE
Geometries outside the hazard domain

### ISSUE THREAD

**Issue by dalmijn** [Deltares/Delft-FIAT#76]
### Delft-FIAT version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have checked that this bug exists on the latest version of Delft-FIAT.

### Reproducible Example

If a geometry falls outside the hazard map domain, it will try to index cells (the overlap between the geometry and the hazard map) outside of the domain (bounding box) which will result in an error.
Basically any exposure vector file suffices as long as a geometry lies outside. 

### Current behaviour

Produces an indexing error...

### Desired behaviour

Handle those geometries properly, don't raise an error. Don't calculate anything and return a message saying that the geometry lies outside of the domain.
